src/utils/translation.ts

```diff
@@ -15,6 +16,7 @@ export const getProTableDictionary = (): IProTableDictionary => ({
     selectAllResults: 'Select all results',
     clear: 'Clear',
   },
+  numberFormat: numberWithCommas,
```

Comment:
this allows to format all numbers in pro table to display digits at the thousands. I leveraged ferlab options.